packages/shared/package.json

```diff
@@ -5,11 +5,13 @@
     "lint:fix": "eslint --fix ."
   },
   "dependencies": {
+    "@urql/exchange-auth": "^2.1.4",
     "eslint": "8.42.0",
     "pino": "^8.14.1",
     "pino-pretty": "^10.0.0",
     "semver": "^7.5.1",
-    "typescript": "5.1.3"
+    "typescript": "5.1.3",
+    "urql": "^4.0.4"
```

Comment:
Not sure - we are importing Client directly in the apps, for types. Also not sure how it would play with gql codegen.
The shared configuration seemed like a rabbithole, decited to take look at it later in separate PR.

Reply:
So please first update urql and then we can think about extraction

maybe dropping next-url will also simplify enough

> Client directly in the apps, for types.

we can technically re-export urql from apps-shared. I don't want to maintain versions in 15 packages